config/config.go

```diff
@@ -484,10 +484,16 @@ type HttpServerOptionsConfig struct {
 	// Enabled WebSockets and server side events support
 	EnableWebSockets bool `json:"enable_websockets"`
 
-	// Deprecated. SSL certificates used by Gateway server.
+  
+	// Deprecated: Use `ssl_certificates`instead.
+	// SSL certificates used by Gateway server.
 	Certificates CertsData `json:"certificates"`
 
-	// SSL certificates used by your Gateway server. A list of certificate IDs or path to files.
+
+	// The certificate store of the Gateway.
```

Comment:
The certificate store is in Redis - I think this is a reference to the certificates (presumably ID is an identifier assigned by Tyk when storing it in Redis?).

My suggestion and other comments assume this to be true.

```suggestion
	// Index of certificates available to the Gateway for use in client and upstream communication.
```